kubernetes-ts-multicloud/README.md

```diff
@@ -0,0 +1,82 @@
+[![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
+
+# Kubernetes Multi-cloud Example
+
+This example creates managed Kubernetes clusters using AKS, EKS, and GKE, and deploys an application
+on each cluster.
+
+## Deploying the App
+
+To deploy your infrastructure, follow the below steps.
+
+### Prerequisites
+
+1. [Install Pulumi](https://www.pulumi.com/docs/reference/install/)
+2. [Install Node.js 8.11.3](https://nodejs.org/en/download/)
+3. (Optional) [Configure AWS Credentials](https://www.pulumi.com/docs/reference/clouds/aws/setup/)
+4. (Optional) [Configure Azure Credentials](https://www.pulumi.com/docs/reference/clouds/azure/setup/)
+5. (Optional) [Configure GCP Credentials](https://www.pulumi.com/docs/reference/clouds/gcp/setup/)
+6. (Optional) [Configure local access to a Kubernetes cluster](https://kubernetes.io/docs/setup/)
+
+### Steps
+
+After cloning this repo, from this working directory, run these commands:
+
+1. Install the required Node.js packages:
+
+    ```bash
+    $ npm install
+    ```
+
+2. Create a new stack, which is an isolated deployment target for this example:
+
+    ```bash
+    $ pulumi stack init
+    ```
+
+3. Set the required configuration variables for this program:
+
+    ```bash
+    $ pulumi config set aws:region us-west-2
+    $ pulumi config set azure:location westus2
+    ```
+   
+   Note that you can choose different regions here.
+
+   We recommend using `us-west-2` to host your EKS cluster as other regions (notably `us-east-1`) may have capacity
+   issues that prevent EKS clusters from creating:
+
+    ```
+    Diagnostics:
```

Comment:
IMO we can leave out the Diagnostics error as the note already calls out capacity issues.